src/components/higlass/HiGlassSubscriber.js

```diff
@@ -111,16 +123,98 @@ export default function HiGlassSubscriber(props) {
   // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [loaders, dataset]);
 
-  const hgComponent = useMemo(() => (
-    <HiGlassComponent
-      zoomFixed={false}
-      viewConfig={hgViewConfig}
-      options={{
-        ...hgOptions,
-        theme,
-      }}
-    />
-  ), [hgViewConfig, hgOptions, theme]);
+  const hgOptions = useMemo(() => ({
+    ...hgOptionsProp,
+    theme,
+  }), [hgOptionsProp, theme]);
+
+  const hgViewConfig = useMemo(() => {
+    // HiGlass needs the start and end absolute genome coordinates
+    const initialXDomain = [
+      genomicTargetX - ((genomeSize / (2 ** genomicZoomX))) * ((width / 2) / HG_SIZE),
+      genomicTargetX + ((genomeSize / (2 ** genomicZoomX))) * ((width / 2) / HG_SIZE),
+    ];
+    const initialYDomain = [
+      genomicTargetY - ((genomeSize / (2 ** genomicZoomY))) * ((height / 2) / HG_SIZE),
+      genomicTargetY + ((genomeSize / (2 ** genomicZoomY))) * ((height / 2) / HG_SIZE),
```

Comment:
Can you comment what each term does in this calculation as well as removing the extra parenthesis around the second term `(genomeSize / (2 ** genomicZoomX))`?